code/modules/food_and_drinks/food/foods/junkfood.dm

```diff
@@ -100,6 +100,35 @@
 	list_reagents = list("nutriment" = 10, "sugar" = 4)
 	tastes = list("syrup" = 3, "waffle" = 2)
 
+/obj/item/reagent_containers/food/snacks/twimsts
+	name = "Twimsts"
+	desc = "Sweet twisted sticks made of liquorice, used for eating and as eatable handcuffs!"
+	icon_state = "twimsts"
+	trash = /obj/item/trash/twimsts
+	antable = FALSE
+	filling_color = "#E31818"
+	junkiness = 5
+	list_reagents = list("nutriment" = 2, "sugar" = 10)
+	tastes = list("sweetness" = 3, "liquorice" = 2)
+
+/obj/item/reagent_containers/food/snacks/twimsts/examine(mob/user)
+	. = ..()
+	. += "<span class='info'>Use in hand to make it into handcuffs.</span>"
```

Comment:
Is it worth repeating this here since the description already indicates that such feature exists?